server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/issue/IssueAssigner.java

```diff
@@ -75,11 +76,15 @@ public void onIssue(Component component, DefaultIssue issue) {
       String scmAuthor = guessScmAuthor(issue);
 
       if (!Strings.isNullOrEmpty(scmAuthor)) {
-        issueUpdater.setNewAuthor(issue, scmAuthor, changeContext);
+        if (scmAuthor.length() <= IssueDto.AUTHOR_MAX_SIZE) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "scmAuthor" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2259)